extensions/dicom-microscopy/src/tools/microscopyManager.js

```diff
@@ -0,0 +1,591 @@
+import Publisher from '../utils/Publisher';
+import ViewerManager, { EVENTS as ViewerEvents } from './viewerManager';
+import RoiAnnotation, {
+  EVENTS as AnnotationEvents,
+} from '../utils/RoiAnnotation';
+import styles from '../utils/styles';
+import { DicomMetadataStore } from '@ohif/core';
+
+const EVENTS = {
+  ANNOTATION_UPDATED: 'annotationUpdated',
+  ANNOTATION_SELECTED: 'annotationSelected',
+  RELABEL: 'relabel',
+  DELETE: 'delete',
+};
+
+/**
+ * MicroscopyManager is responsible to manage multiple third-party API's
+ * microscopy viewers expose methods to manage the interaction with these
+ * viewers and handle their ROI graphics to create, remove and modify the
+ * ROI annotations relevant to the application
+ */
+class MicroscopyManager extends Publisher {
```

Comment:
this service (manager) should get registered to servicesManager. Look at out other extension's services for that. For instance see `ToolGroupService` there is a REGISTRATION that should be added to your manager here (I recommend calling it service). Then you can grab its instance from servicesManager.services throughout the app